Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -730,6 +731,20 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
             }
             else if self.isHighlightPerDragEnabled
             {
+                if
+                    let marker,
+                    let lastHighlighted = highlighted.first,
+                    let data,
```

Comment:
so is it lastHighlighter or firstHighlighted?

Reply:
Unfortunately, lastHighlighted property sometimes isn't set so we take the first highlight